docs/features/software-templates/input-examples.md

```diff
@@ -349,20 +374,24 @@ Testing of this functionality is not yet supported using _create/edit_. In addit
 
 :::
 
+Its possible to use placeholders to reference remote files. This is useful when you have some standard parameters or actions that you want to reuse across multiple templates.
```

Comment:
Please LMK if I understood what this is doing correctly, I am just making assumptions based on what it looks like its doing 😅